### DISCUSSION TITLE
Packet mark

### DISCUSSION THREAD

**Discussion by shigabutdinoff** [hufrea/byedpi#123] [Ideas]
Добавить маркировку пакетам, которые были запутаны и не запутаны

**Comment by hufrea** [comment]
Зачем?

**Comment by shigabutdinoff** [comment]
Чтобы на маршрутизаторе добавить логику с обработанным трафиком

**Comment by shigabutdinoff** [comment]
Допустим такой трафик, я мог основываясь на маркировке пакета завернуть в vpn соединение

**Comment by ko22009** [comment]
> Допустим такой трафик, я мог основываясь на маркировке пакета завернуть в vpn соединение

bolvan-zapret, там не знаешь, есть такая возможность? я бы тоже хотел получить такое решение, чтобы в vpn соединение заворачивать
